frontend/src/packages/auth/auth-api.ts

```diff
@@ -35,6 +36,19 @@ class AuthApi extends BaseHttpApi {
 
     return await response.json<UserSignUpResponseDto>();
   }
+
+  public async getUser(): Promise<UserGetAllItemResponseDto> {
```

Comment:
Let's create a new type in shared, for now it will have the same interface, but we will change it in Sign Up #14 

```suggestion
  public async getAuthenticatedUser(): Promise<UserAuthResponseDto> {
```